meta/mar-ke/themore/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "mar-ke/themore",
+  "description": "themore",
+  "homepage": "https://themore.mar-ke.de",
+  "keywords": [
+    "contao",
+    "mar-ke",
+    "themore",
+    "theme"
+  ],
+  "type": "contao-theme",
+  "license": "proprietary",
+  "authors": [
+    {
+      "name": "Digitalagentur mar_ke",
+      "homepage": "https://mar-ke.de"
+    }
+  ],
+  "support": {
+    "homepage": "https://mar-ke.de"
```

Comment:
This key is actually incorrect and should be removed. See https://getcomposer.org/doc/04-schema.md#support for the valid options.